beniget/beniget.py

```diff
@@ -202,6 +202,15 @@ def _str(self, nodes):
 # this should probably be assigned to the filename give to DefUseChains instead.
 Builtins["__file__"] = __file__
 
+# Cope with conditionally existing builtins by special-casing them.
+Builtins.setdefault('WindowsError', OSError) # never defined under Linux
```

Comment:
I like the idea, but I 'm not fine with the introduction of aliasing with existing objects, which can lead to confusion in client code. Could you just create a different `object()` for each default?